.github/workflows/go-lint.yaml

```diff
@@ -7,6 +7,7 @@ on:
       - ".github/workflows/go-lint.yaml"
       - "cli/**"
       - "infra/blueprint-test/**"
+      - "tflint-ruleset-blueprint/**"
 permissions:
   contents: read
 
```

Comment:
Can't comment on line 25, but need to add `tflint-ruleset-blueprint`

Reply:
Thanks! Missed that, added now!